python/scripts/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+from generate_holidays_file import generate_holidays_file
+
+generate_holidays_file()
```

Comment:
Can you show me how you run this script from the command line?  I think ideally this is just added to the `generate_holidays_file.py` in a block at the bottom:

```
if __name__ == "__main__":
    # execute only if run as a script
    generate_holidays_file()
```